### ISSUE TITLE
Ability to Halt Scan in Main UI

### ISSUE THREAD

**Issue by chryton** [manyfold3d/manyfold#2196]
**Is your feature request related to a problem? Please describe.**
Not per se

**Describe the solution you'd like**
Clicking on "Scanning" while scanning should offer the opportunity to halt the scan.

**Describe alternatives you've considered**
If not able to halt, giving some sort of status on progress would be helpful (thinking of what Plex shows while scanning) as having a large collection of files doesn't offer any feedback as to how long the initial scan will take or offer any affordance to stop if you realize you need to change settings.

**Additional context**
Thinking especially for new users testing it out and also would be a nice affordance for contributors while testing.

**Comment by Floppy** [comment]
Yep, definitely agree. The recently-added activity log is part of giving more feedback on how long it might take, but there's more that could be done.

**Comment by Floppy** [comment]
All scanning activity happens on the `scan` queue, so in a sense, clearing that queue would do the job to stop the scan, but the folder scan can take a long time too, and that job would need to be actually stopped partway through. Perhaps anything running on the `scan` queue could be halted.
